sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -31259,7 +31259,7 @@
                   "tableView": false,
                   "inputType": "text",
                   "inputMask": "",
-                  "label": "Please estimate your weekly transportation costs.",
+                  "label": "Please estimate your weekly transportation costs:",
```

Comment:
In the PDF is says this question should be removed.

Reply:
There's no AC for that in the ticket. Please just revert the changes in that line.

Reply:
Updated